src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -92,10 +94,10 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
   implicit val eth_getTransactionByBlockHashAndIndex =
     new JsonDecoder[GetTransactionByBlockHashAndIndexRequest] with JsonEncoder[GetTransactionByBlockHashAndIndexResponse] {
       override def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetTransactionByBlockHashAndIndexRequest] = params match {
-        case Some(JArray((blockHash: JString) :: (transactionIndex: JString) :: Nil)) =>
+        case Some(JArray(JString(blockHash) :: transactionIndex :: Nil)) =>
```

Comment:
Geth allows you to specify numbers both a hex strings, and as regular JSON numbers (decimal). I don't see why we would deprive ourselves of such convenience.